app/controllers/mixins/containers_common_mixin.rb

```diff
@@ -41,8 +41,16 @@ def button
     end
   end
 
+  def show_list
+    process_show_list
+  end
+
   private
 
+  def display_name
+    ui_lookup(:models => @record.class.base_class.name)
```

Comment:
> imo it's a reasonable change. there's no other replicators in the system (unlike, for example, services), so it doesn't sound like it will cause a confusion.

@abonas sorry, perhaps I didn't explain it well, for every affected entity this changes "Container <Entity>" to "<Entity>". 
So, Services, Images, Image Registries and Nodes are also changed. Do you have a suggestion?